config.json

```diff
@@ -54,7 +54,7 @@
     "homepage_featured_products_count": 4,
     "homepage_top_products_count": 4,
     "homepage_show_carousel": true,
-    "homepage_show_carousel_arrows": true,
+    "homepage_show_carousel_arrows": false,
```

Comment:
Why do we need this change?

Reply:
thanks for paying attention. It's been made for manual testing and left it by mistake. turn to back.